tedge/src/cli/connect/common_mosquitto_config.rs

```diff
@@ -24,6 +25,7 @@ impl Default for CommonMosquittoConfig {
                 "subscribe".into(),
                 "unsubscribe".into(),
             ],
+            message_size_limit: "256MB".into(),
```

Comment:
The mqtt payload size set to 256MB by default is it required to add it here as well?

Reply:
Just thought that it better to add because we know whats the default value is.